src/main/drivers/camera_control.c

```diff
@@ -0,0 +1,158 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "camera_control.h"
```

Comment:
Thanks, I had a question about VTX_CONTROL, why would a translation unit do conditional compilation if it will be omitted at the linking phase :-)
Seems like this is the reason. Ok, I'll add an `#ifdef` clause.